src/main/resources/application.yml

```diff
@@ -21,3 +21,22 @@ nldi:
   displayProtocol: ${nldiProtocol}
   displayHost: ${nldiHost}
   displayPath: ${nldiPath:}
+
```

Comment:
This renames the Actuator endpoints to hide them and only allows the heath and info ones.